src/flow.h

```diff
@@ -76,6 +77,8 @@ typedef struct AppLayerParserState_ AppLayerParserState;
 #define FLOW_TOCLIENT_DROP_LOGGED         0x00004000
 /** alproto detect done.  Right now we need it only for udp */
 #define FLOW_ALPROTO_DETECT_DONE          0x00008000
+/** time machine is enabled on this flow */
+#define FLOW_TIMEMACHINE_ENABLED          0x00010000
```

Comment:
Correct, packets should be dumped to disk rather than added to timemachine so this may be a bit misleading terminology wise.  Maybe FLOW_SKIP_TIMEMACHINE would be a better name.